http/src/main/kotlin/io/micronaut/http/bind/binders/ContinuationArgumentBinder.kt

```diff
@@ -50,13 +50,16 @@ class ContinuationArgumentBinder : TypedRequestArgumentBinder<Continuation<*>> {
         private val reactorContextPresent: Boolean = ClassUtils.isPresent("kotlinx.coroutines.reactor.ReactorContext", null);
 
         @JvmStatic
-        fun setupCoroutineContext(source: HttpRequest<*>, contextView: ContextView) {
+        fun setupCoroutineContext(source: HttpRequest<*>, contextView: ContextView, continuationArgumentBinderCoroutineContextFactories: Collection<HttpCoroutineContextFactory<*>>) {
```

Comment:
I believe this is public correct? If so then this is a breaking change

Reply:
Denis is probably gone for the day. A simple revision could be to add another method with overloaded arguments. This class should really have been `@Internal`